test/inductor/test_torchinductor_dynamic_shapes.py

```diff
@@ -912,6 +912,18 @@ def f(x):
 
         f(torch.tensor([5], device=device))
 
+    @torch._dynamo.config.patch(capture_scalar_outputs=True)
+    def test_symint_sum_list(self, device):
+        @torch.compile()
+        def f(xt):
+            xs = xt.tolist()
+            for x in xs:
+                torch._check_is_size(x)
+            y = sum(xs)
```

Comment:
how does inductor lowering the `sum(List[unbacked symint])` actually end up testing the `statically_known_multiple_of` code?

(i can't think of a case where inductor will be dividing two unbacks symints given this repro)

Reply:
It's not the sum, it's actually the torch.empty call lol